data/nav.yml

```diff
@@ -429,7 +429,30 @@
           path: "apis/rest-api"
         - name: "Limits"
           path: "apis/rest-api/limits"
-        - name: "Pipelines REST API"
+        - name: "Agent"
+          children:
+            - name: "Overview"
+              path: "apis/agent-api"
+            - name: "Metrics"
+              path: "apis/agent-api/metrics"
+        - name: "Analytics"
+          children:
+            - name: "Flaky tests"
+              path: "apis/rest-api/analytics/flaky-tests"
+            - name: "Runs"
+              path: "apis/rest-api/analytics/runs"
+            - name: "Suites"
+              path: "apis/rest-api/analytics/suites"
+            - name: "Tests"
+              path: "apis/rest-api/analytics/tests"
+        - name: "Organizations"
+          children:
+            - name: "Organizations"
+              path: "apis/rest-api/organizations/organizations"
```

Comment:
I suspect using this repeated URL slug pattern might be causing part of the problem I described in my first comment of this PR:
`path: "apis/rest-api/organizations/organizations"`

It's been used a few other times in this PR.